.gitignore

```diff
@@ -25,3 +25,6 @@ pip-log.txt
 
 #Mr Developer
 .mr.developer.cfg
+
+#Pycharm IDE settings
```

Comment:
Can we drop this, please? Putting this in your individual `~/.gitignore` ensures that we don't need to account for all possible IDEs. While you're at it, could you drop the `.mr.developer.cfg` line, too?